handout/routing/README.md

```diff
@@ -1,3 +1,3 @@
 # Routing
 
-In this section we will discuss the role of routing in Single Page Applications and Angular 2's new component router.
+In this section we will discuss the role of routing in single page aplications and Angular 2's new component router.
```

Comment:
most places seem to use Single Page Application, or Single-Page Application, or SPA as the acronym.